lib/flight_job/models/job/adjust_active_index.rb

```diff
@@ -34,17 +34,29 @@ class Job < ApplicationModel
     #
     # XXX The active index file may also be updated when a job is saved.
     # Managing that should also become the responsibility of this class.
-    class AdjustActiveIndex
+    class AdjustActiveIndex < Job
```

Comment:
Now I understand.  `edit_active_index` should be defined as a class method instead of an instance method.  It can then be called as simply `edit_active_index` instead of `job.edit_active_index`.  Then this class will not need to inherit from `Job`.